client/src/grpc.rs

```diff
@@ -0,0 +1,346 @@
+/// QueryFlightRequest
+#[derive(Eq)]
+#[derive(Clone, PartialEq, ::prost::Message)]
+pub struct QueryFlightRequest {
+    /// is_cargo - true if cargo mission, false if people transport
+    #[prost(bool, tag="1")]
+    pub is_cargo: bool,
+    /// persons - number of people for transport
+    #[prost(uint32, optional, tag="2")]
+    pub persons: ::core::option::Option<u32>,
+    /// weight in grams
+    #[prost(uint32, optional, tag="3")]
+    pub weight_grams: ::core::option::Option<u32>,
+    /// requested preferred time of flight (if omitted - asap)
+    #[prost(message, optional, tag="4")]
+    pub requested_time: ::core::option::Option<::prost_types::Timestamp>,
```

Comment:
Separate requested arrival time and departure time as optional fields -
- cargo shipments generally are scheduled by arrival time
- rideshares are generally scheduled by departure time

Reply:
I updated the impl to accept either departure or arrival time now. At least one is requied.